docs/BUILD.md

```diff
@@ -167,12 +187,12 @@ This is the rootfs filesystem in a tar file.
 It is a temporary artifact which is used as input into security and bill-of-materials scanners,
 and is then consumed to create other artifacts, and then can be removed. To build it:
 
-1. Verify the existence of the linuxkit builder configuration file `images/rootfs.yml`. See notes on [generating yml](#generating-yml).
+1. Verify the existence of the linuxkit builder configuration file `images/rootfs.yml`. See notes on [generating yml](#generating-yml) to understand how this is generated.
 1. Call `makerootfs.sh tar -y images/rootfs.yml -t path/to/rootfs.tar [-a <arch>]`, which will build an image for the target architecture `<arch>` using `linuxkit` with a tar output format using `images/rootfs.yml` as the configuration file, saving the output to `path/to/rootfs.tar`.
 
 When done with any later builds, you can remove the temporary artifact `path/to/rootfs.tar`.
 
-When run with `make rootfstar` or `make rootfs`, this will build the rootfs tar file:
+When run with `make rootfstar` or `make rootfs.tar` or `make rootfs`, this will build the rootfs tar file:
```

Comment:
Missing period in rootfstar? Oh, I see it does work with rootfstar. Is this intentionally or by accident?

Reply:
Intentional. I remember vaguely (and could be wrong) that people kept typing it without the `.` by accident, and we just added it for convenience.